core/src/actor.rs

```diff
@@ -348,4 +422,23 @@ mod tests {
             )
             .unwrap();
     }
+
+    #[tokio::test]
+    async fn derive_winternitz_pk() {
```

Comment:
Let's also add one more test that has a hardcoded public key

Reply:
Fixed in https://github.com/chainwayxyz/clementine/pull/371/commits/ec074f4c5b2bc7385c14f4a41f4e775c3fdc519f.